cnexp_array/neuron/leonhard.cpp

```diff
@@ -165,6 +183,7 @@ namespace neuron {
     void nrn_init_leonhard(_nrn_model_sorted_token const& _sorted_token, NrnThread* _nt, Memb_list* _ml_arg, int _type) {
         _nrn_mechanism_cache_range _lmr{_sorted_token, *_nt, *_ml_arg, _type};
         auto inst = make_instance_leonhard(_lmr);
+        auto node_data = make_node_data_leonhard(*_nt, *_ml_arg);
```

Comment:
This is one of the issue NMODL has, I think it can't know if the voltage will be used, hence it just prints the line unconditionally an prays the compiles removes it.